### DISCUSSION TITLE
Media Markt , Saturn fix Availability

### DISCUSSION THREAD

**Discussion by dacamel** [jef/streetmerchant#2325] [Ideas and feature requests]
The store info for Media Markt and Saturn is not correct.

Availability can be checked by looking up each product site and checking the source code for

{"__typename":"GraphqlDeliveryTime","availabilityType":

If some products are avaible, the status changes from Quantity 0 to 1

https://abload.de/img/mmsldkc2.jpg

If some one can add this to  streetmerchant i would pay also for it : )

**Comment by web-josh** [comment]
Can you provide more information?

-How does this work with the basket trick?
-When Quantity changes from 0 to 1 in source does the add to cart button becomes green?
-When Quantity changes from 0 to 1 in source does the add to cart button stays gray but product is buyable when already in cart?

This should be pretty simple to fix but I need more infos/an explanation as I have not seen it live on those pages.

Feel free to pm me in german (assuming you are german :P)
